packages/web/src/scss/tools/_tokens.scss

```
@@ -9,3 +11,13 @@
         }
     }
 }
+
+@function get-variable-if-exists($variable-name, $fallback-value) {
+    @if map.has-key(meta.module-variables(tokens), $variable-name) {
+        @return map.get(meta.module-variables(tokens), $variable-name);
+    } @else {
+        @debug 'Token not found: #{ $variable-name }';
```

```suggestion
        @debug 'Token not found: #{$variable-name}';
```

🤷 what do you think?